### PR TITLE
Footnote improvements

### PR DESCRIPTION
**Pull request by Integral**
- Add empty paragraph on new footnote creation (#707) 
- Remove footnotes from metadata editor (#706)

### CODE REVIEW

**Comment by Integral** [comment]
@michael could you take a look?

File: src/article/shared/FootnoteComponent.js [hunk 1]
```diff
@@ -37,6 +39,21 @@ export default class FootnoteComponent extends Component {
         )
       )
     }
+
+    if (mode === METADATA_MODE) {
+      const footer = $$('div').addClass('se-footer').append(
+        $$('button').addClass('se-remove-item').append(
```
Comment (oliver----) [inline src/article/shared/FootnoteComponent.js]: Please use the `Button` component.

File: src/article/shared/FootnoteComponent.js [hunk 2]
```diff
@@ -37,6 +39,21 @@ export default class FootnoteComponent extends Component {
...
+          $$(FontAwesomeIcon, { icon: 'fa-trash' }).addClass('se-icon'),
+          'Remove'
```
Comment (oliver----) [inline src/article/shared/FootnoteComponent.js]: Please start using `this.getLabel()`.
Brother of `CopyNPaste` is `StringLiterals`.

**Comment by oliver----** [comment]
@Integral please rebase, I have merged some changes in the meantime.

File: src/article/shared/FootnoteComponent.js (hunk 2 above)
Comment (Integral) [reply src/article/shared/FootnoteComponent.js]: 👍

**Comment by Integral** [comment]
@oliver---- i've changed what you said, thanks

**Comment by oliver----** [comment]
Great. Thanks!

**Comment by oliver----** [comment]
You can merge :)

**Comment by oliver----** [comment]
Ah wait.

File: src/article/shared/FootnoteComponent.js [hunk 3]
```diff
@@ -37,6 +39,22 @@ export default class FootnoteComponent extends Component {
         )
       )
     }
+
+    if (mode === METADATA_MODE) {
+      const Button = this.getComponent('button')
+      const footer = $$('div').addClass('se-footer').append(
+        $$(Button).addClass('se-remove-item').append(
```
Comment (oliver----) [inline src/article/shared/FootnoteComponent.js]: Try to use as much of the component props as u need, e.g. in this case you could provide icon, and theme.
https://github.com/substance/texture/blob/master/src/kit/ui/Button.js#L3-L25

**Comment by Integral** [comment]
ok, done

**Comment by Integral** [comment]
ok, done

**Comment by oliver----** [comment]
Now let's merge.

**Pull request merged**